video/clip.go

```diff
@@ -55,6 +62,13 @@ func getRelevantSegment(allSegments []*m3u8.MediaSegment, playHeadTime float64,
 	return 0, fmt.Errorf("error clipping: did not find a segment that falls within %v seconds", playHeadTime)
 }
 
+// Function that will take a source URL manifest and return a new URL
+// pointing to the clipped manifest
+func ClipInput(requestID string, srcUrl *url.URL, startTime, endTime float64) (*url.URL, error) {
```

Comment:
Is the plan for this function to call ClipManifest after parsing `srcUrl` into a manifest?

Reply:
Yep, using a separate PR and trying to keep PRs small for easier review.